implementation/eNotice/owl_ontology/eNotice.rdf

```diff
@@ -49,17 +48,17 @@
       <rdfs:label xml:lang="en">eProcurement Ontology Notice - core</rdfs:label>
       <dct:description xml:lang="en">The eProcurement Ontology Notice core describes the concepts and properties representing the European Public Procurement Notice domain. The provision of these semantics offers the basis for a common understanding of the domain for all stakeholders ensuring the quality of data exchange and transparency. The ontology restrictions are published in a separate artefact.</dct:description>
       <dct:publisher>http://publications.europa.eu/resource/authority/corporate-body/PUBL</dct:publisher>
-      <rdfs:comment>This version is automatically generated from eNotice.xml on 2023-11-09</rdfs:comment>
+      <rdfs:comment>This version is automatically generated from eNotice.xml on 2024-01-29</rdfs:comment>
       <rdfs:seeAlso rdf:resource="https://github.com/OP-TED/ePO/releases"/>
       <rdfs:seeAlso rdf:resource="https://joinup.ec.europa.eu/collection/eprocurement/solution/eprocurement-ontology/about"/>
       <rdfs:seeAlso rdf:resource="https://op.europa.eu/en/web/eu-vocabularies/e-procurement"/>
       <rdfs:seeAlso rdf:resource="https://docs.ted.europa.eu/home/index.html"/>
-      <dct:issued rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2023-11-09</dct:issued>
+      <dct:issued rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2024-01-29</dct:issued>
       <dct:created rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2021-06-01</dct:created>
-      <owl:versionInfo>4.0.0</owl:versionInfo>
+      <owl:versionInfo>4.0.1</owl:versionInfo>
       <owl:incompatibleWith>3.1.0</owl:incompatibleWith>
-      <owl:versionIRI rdf:resource="http://data.europa.eu/a4g/ontology#not-4.0.0"/>
-      <owl:priorVersion>http://data.europa.eu/a4g/ontology#not-3.1.0</owl:priorVersion>
+      <owl:versionIRI rdf:resource="http://data.europa.eu/a4g/ontology#not-4.0.1"/>
```

Comment:
question, for me to know. What is the 'not' part of 
http://data.europa.eu/a4g/ontology#not-3.1.0 ? it stands for what?

Reply:
it comes from the "epo-not" prefix used for the eNotice module.